   ml_ops/sm-datazone_import/README.md

```diff
@@ -31,11 +32,43 @@ Run the script and follow the instructions.
 ```bash
 python import-sagemaker-domain.py \
     --region REGION \
-    --federation-role ARN_OF_FEDERATION_ROLE \
     --account-id ACCOUNTID
 ```
 
 ### Additional Configuration
 
 - SageMaker execution roles need DataZone API permissions in order for the Assets UI to function. See [DataZoneUserPolicy.json](./resources/DataZoneUserPolicy.json) for an example.
-- Ensure the DataZone Domain trusts SageMaker. In the AWS DataZone console navigate to Domain details and select the "Trusted services".
\ No newline at end of file
+- Ensure the DataZone Domain trusts SageMaker. In the AWS DataZone console navigate to Domain details and select the "Trusted services".
+
+### Potential errors and workarounds
+
+**Cannot view ML assets in SageMaker Studio**
+
+Make sure that the execution role that is attached to the SageMaker User in the attached domain has ListTags attached as a permissions policy to the role. A simple workaround is to attach AmazonSageMakerCanvasFullAccess policy which contains this permission. Without it - you will not be able to view the Assets tab in the Studio UI. Expected error in the UI inspect: 
+```
+User: arn:aws:sts::789706018617:assumed-role/AmazonSageMaker-ExecutionRole-20241127T120959/SageMaker is not authorized to 
+perform: sagemaker:ListTags on resource: arn:aws:sagemaker:us-east-1:789706018617:domain/d-qy9jzu4s7q0y because no 
+identity-based policy allows the sagemaker:ListTags action
+```
+
+**Able to view assets in sidebar, but page is not loading.**
+
+If you are able to view the assets - but are getting a There was a problem when loading subscriptions error in the page where your ML assets should be - ensure that the SageMaker Execution role tied to this SageMaker user has AmazonDataZoneFullAccess or a more limited AmazonDataZoneFullUserAccess attached to it. 
+
+**DataZone portal is not showing a generated action-link for user.**
+
+If you are attempting to create ProjectB using a subset of users B under created environment B - make sure. that you use a separate federation role when the _associate_fed_role action is called. This is required or else the association will fail and thus the subsequent call to create_environment_action will fail with the following error. 
+See `../resources` for sample permissions and trust policies for the federation role. Be sure to fill in your SageMaker Domain Id.
+
+```
+An error occurred (ValidationException) when calling the AssociateEnvironmentRole operation: Role Arn 
+arn:aws:iam::789706018617:role/svia-test-byod-fed-role already being used in a different project
+```
+
+Successful association will return the following
```

Comment:
is there some way user can verify federation role after running the script?

Reply:
Verify the role has the correct permissoins or verify that it will work via console? The latter they can simple open the UI link in console and if it allows them to open the Studio tab, then it is correct. I can add this note as af ollow up.